packages/search/src/searchProject.ts

```diff
@@ -53,13 +75,14 @@ export function* searchProject({
           memo,
         },
         state,
-      )
+        options: options as any,
```

Comment:
I tried in various ways to make Typescript happy, but this seemed like the easiest fix. Even though I use function overloading, it would complain about union types when I called `visitNode`.